src/auto_scheduler/search_task.cc

```diff
@@ -66,6 +69,13 @@ HardwareParams HardwareParamsNode::GetDefaultHardwareParams(const Target& target
 
     device_api->GetAttr(ctx, tvm::runtime::DeviceAttrKind::kMaxRegistersPerBlock, &ret);
     int max_registers_per_block = ret;
```

Comment:
I think this is a bug.
I will send another PR to rename `max_registers_per_block` to `max_local_memory_per_block` to make it align with `VerifyGPUCode` pass.

Reply:
This check should not be used in my option. Autotvm does not use this check either.